openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -269,7 +269,7 @@ The following diagram is based on the Credential Issuer initiated flow, as illus
         |   interacts    |                                                    |
         |--------------->|                                                    |
         |                |                                                    |
-        |                |  (4) Token Request (Pre-Authorized Code, pin)      |
+        |                |  (4) Token Request (Pre-Authorized Code, tx_code)  |
```

Comment:
Should this be in a separate PR?

Reply:
I think this is a left-over, might be addressed by existing PRs

Reply:
this happened because the branch used in Paul's PR was not up to date with main. will make sure there are no conflicts when merging.